### PR TITLE
Smoke test Endpoint: Account Memberships

### PR DESCRIPTION
**Pull request by RogerAlvarezM**

### CODE REVIEW

File: Pivotal/features/project/projects_project_memberships.feature [hunk 1]
```diff
@@ -11,4 +11,6 @@ Feature: Get account
     When I send the request
     Then I get a "200" status code as response
 
+
```
Comment (estigmata) [inline Pivotal/features/project/projects_project_memberships.feature:14]: Should be only one empty line end of file

File: Pivotal/features/project/project_acceptace.feature [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-      And I set up the data
-      """
-      {
-      "Content": "API Test",
-      "Icon": 4
-      }
-      """
-    When I send the request
-    Then I get a "200" status code as response
+#@acceptance
```
Comment (jorgeflores742) [inline Pivotal/features/project/project_acceptace.feature]: delete the character #

Comment (estigmata) [inline Pivotal/features/project/project_acceptace.feature]: Should did not have code commented

File: Pivotal/environment.yml [hunk 3]
```diff
@@ -1,3 +1,4 @@
 api_url: https://www.pivotaltracker.com/services/v5
 user: team2at08@mailinator.com
 pass: at08team2
+token: d1b81ba9bc260c5a832872933fe2d39d
```
Comment (estigmata) [inline Pivotal/environment.yml:4]: Please add one empty file end of file

File: Pivotal/features/project/project_smoke.feature [hunk 4]
```diff
@@ -1,4 +1,4 @@
-@smoke
+#@smoke
```
Comment (jorgeflores742) [inline Pivotal/features/project/project_smoke.feature]: why you added the character # ?

**Pull request merged**